### ISSUE TITLE
Proposal : Flutter Inspector Integration

### ISSUE THREAD

**Issue by shibupanda** [truongsinh/appium-flutter-driver#56]
## Description:
There is no such tool available now that can retrieve flutter finders. 
Need to build an inspector it may integrate with Appium Desktop or Maybe an independent tool which can inspect element for a flutter application.

## Problem:
Now user requires the source code to be there in IntelliJ editor to inspect the element.
In some of the cases it quite difficult to get the complete source code and there they stuck.

**Comment by shibupanda** [comment]
cc @truongsinh

**Comment by truongsinh** [comment]
Are you refering to https://flutter.dev/docs/development/tools/devtools/inspector? Is it related to #36 or https://api.flutter.dev/flutter/flutter_driver/FlutterDriver/getRenderObjectDiagnostics.html or https://api.flutter.dev/flutter/flutter_driver/FlutterDriver/getRenderTree.html

**Comment by shibupanda** [comment]
Yes, @truongsinh I am referring to https://flutter.dev/docs/development/tools/devtools/inspector.
Is there any possibility we can integrate it to Appium Desktop or any independent tool.

**Comment by truongsinh** [comment]
I am not extensively using Appium Desktop, and not fimilar with its API, but I think we can start small, with #36 , and goes from there.

**Comment by shibupanda** [comment]
Sure. Thanks for the clarification.

**Comment by varaprasad-pakalapati** [comment]
Hi,

any update on this?
